to_be_ccppized/shr_orb_mod.F90

```diff
@@ -0,0 +1,141 @@
+! Copyright (C) 2024 National Science Foundation-National Center for Atmospheric Research
+! SPDX-License-Identifier: Apache-2.0
+module shr_orb_mod
```

Comment:
I suppose this is fine since it's a temporary solution but I wonder if the copy of `shr_orb_mod` in CAM-SIMA's `share` submodule can be used directly here, perhaps through a dependency (`dependencies = ../../../../../share/shr_orb_mod.F90` in the ccpp metadata. Just wanted to open it up for discussion in case it's an easy fix. Please don't hold up the PR if not, I think this is best computed in the host model and passed as a CCPP variable eventually (e.g., `cosine_of_solar_zenith_angle`)